src/main/java/talkabot/task/Task.java

```diff
@@ -0,0 +1,63 @@
+package talkabot.task;
+
+/**
+ * Task class contains task description and completion status.
+ */
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    /**
+     * Constructor for Task class.

```

Comment:
Same constructor issue for this, I won't be commenting on this anymore, maybe you can check the other classes regarding this issue as well